docs/user-guide/dpo.rst

```diff
@@ -2,17 +2,23 @@
 
 .. _model-aligner-dpo:
 
-Model Alignment by Direct Preference Optimization (DPO)
-@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+Model Alignment by DPO, RPO and IPO
```

Comment:
fix punctuation.

Model Alignment by DPO, RPO, and IPO

Reply:
```suggestion
Model Alignment by DPO, RPO, and IPO
```